beetsplug/acousticbrainz.py

```diff
@@ -41,64 +45,168 @@ def func(lib, opts, args):
+                log,
+                high_data,
+                ["highlevel", "gender", "value"],
+            )
+            item.genre_rosamerica = get_value(
+                log,
+                high_data,
+                ["highlevel", "genre_rosamerica", "value"],
+            )
+            item.mood_acoustic = get_value(
+                log,
+                high_data,
+                ["highlevel", "mood_acoustic", "all", "acoustic"],
+            )
+            item.mood_aggressive = get_value(
+                log,
+                high_data,
+                ["highlevel", "mood_aggresive", "all", "aggresive"],
+            )
+            item.mood_electronic = get_value(
+                log,
+                high_data,
+                ["highlevel", "mood_electronic", "all", "electronic"],
+            )
             item.mood_happy = get_value(
                 log,
-                data,
+                high_data,
                 ["highlevel", "mood_happy", "all", "happy"],
             )
             item.mood_party = get_value(
                 log,
-                data,
+                high_data,
                 ["highlevel", "mood_party", "all", "party"],
             )
+            item.mood_relaxed = get_value(
+                log,
+                high_data,
+                ["highlevel", "mood_relaxed", "all", "relaxed"],
+            )
+            item.mood_sad = get_value(
+                log,
+                high_data,
+                ["highlevel", "mood_sad", "all", "sad"],
+            )
+            item.rhythm = get_value(
+                log,
+                high_data,
+                ["highlevel", "ismir04_rhythm", "value"],
+            )
+            item.tonal = get_value(
+                log,
+                high_data,
+                ["highlevel", "tonal_atonal", "all", "tonal"],
+            )
+            item.voice_instrumental = get_value(
+                log,
+                high_data,
+                ["highlevel", "voice_instrumental", "value"],
+            )
+            item.average_loudness = get_value(
+                log,
+                low_data,
+                ["lowlevel", "average_loudness"],
+            )
+            item.chords_changes_rate = get_value(
+                log,
+                low_data,
+                ["tonal", "chords_changes_rate"],
+            )
+            item.chords_key = get_value(
+                log,
+                low_data,
+                ["tonal", "chords_key"],
+            )
+            item.chords_number_rate = get_value(
+                log,
+                low_data,
+                ["tonal", "chords_number_rate"],
+            )
+            item.chords_scale = get_value(
+                log,
+                low_data,
+                ["tonal", "chords_scale"],
+            )
+            item.key_key = get_value(
+                log,
+                low_data,
+                ["tonal", "key_key"],
+            )
```

Comment:
For this one, we actually have a field name already: `initial_key`. https://github.com/sampsyo/beets/blob/master/beets/library.py#L449

We can use this if the naming format looks similar enough. If not, then we might be stuck using a separate field.